posthog/hogql/metadata.py

```diff
@@ -152,7 +152,7 @@ def visit_cte(self, node: ast.CTE):
 
     def visit_join_expr(self, node: ast.JoinExpr):
         if isinstance(node.table, ast.Field):
-            self.table_names.add(node.table.chain[0])
+            self.table_names.add(".".join([str(x) for x in node.table.chain]))
```

Comment:
nit: feels like it would be useful to have some kind of helper methods for things like this